index.html

```diff
@@ -151,18 +151,18 @@ <h2 class="action-head">Take <span class="red">Action</span>:</h2>
 							</div>
 						</div>
 						<div class="wrapper wrapper-space">
-							<h3 id="list">Join Our <span class="red">Mailing List</span></h2>
+							<h2 id="list">Join Our <span class="red">Mailing List</span></h2>
```

Comment:
Updated this to match the closing tag, since visually I think that is what we wanted this to be.

Reply:
if we are going to do that, could we also change the `<h3>More From <span class="red">TBFighters</span> and TB Advocacy</h3>` to an `h2` because it isn't "under" the mailing list

Reply:
Ahh good call out, I'll change it to a h3, it should probably be under the "Take Action" h2!